.github/ISSUE_TEMPLATE/hackathon-project-template.md

```diff
@@ -1,9 +1,96 @@
+**public** channel with one of the following template names:
+
+- hbmhack-NAME_OF_YOUR_PROJECT
+- hbm-NAME_OF_YOUR_PROJECT
+
+These would be the corresponding URLs that you can paste here.
+
+https://mattermost.brainhack.org/brainhack/channels/hbmhack-NAME_OF_YOUR_PROJECT
+https://mattermost.brainhack.org/brainhack/channels/hbm-NAME_OF_YOUR_PROJECT
+-->
+
+<!--
+**Video channel**:
+
+We are trying to be super careful about "zoom bombing" possibility.
+So we want to avoid having links to video chats in "public space".
+We suggest that you create a Jitsi or Zoom room and mention it in your text channel as "pinned" message or in the channel header.
+
+-->
+
+**Image for the OHBM brainhack website**
+
+## Project submission
+
+## Submission checklist
+*Once the issue is submitted, please check items in this list as you add under 'Additional project info'*
+
+Please include the following above (all required):
+-   [ ] Link to your project: could be a code repository, a shared document, etc. See [here](https://github.com/ohbm/hackathon2021/blob/master/.github/ISSUE_TEMPLATE/handbooks/projects.md#link-to-project)
+-   [ ] Include your [Mattermost handle](https://mattermost.brainhack.org/) (i.e. your username). If you do not have an account, please [sign up here](https://mattermost.brainhack.org/signup_email).
+-   [ ] Goals for the OHBM Brainhack: describe what you want to achieve during this brainhack. See [here](https://github.com/ohbm/hackathon2021/blob/master/.github/ISSUE_TEMPLATE/handbooks/projects.md#goals).
+-   [ ] Flesh out at least 2 "good first issues": those are tasks that do not require any prior knowledge about your project, could be defined as issues in a GitHub repository, or in a shared document, cf [here](https://github.com/ohbm/hackathon2020/blob/master/.github/ISSUE_TEMPLATE/handbooks/projects.md#onboarding-2-good-first-issues).
+-   [ ] Skills: list skills that would be particularly suitable for your project. We ask you to include at least one non-coding skill, cf. [here](https://github.com/ohbm/hackathon2021/blob/master/.github/ISSUE_TEMPLATE/handbooks/projects.md#onboarding-skills).
+-   [ ] Chat channel: A link to a chat channel that will be used during the OHBM Brainhack. This can be an existing channel or a new one. We recommend using the [Brainhack space on mattermost](https://mattermost.brainhack.org/), cf. [here](https://github.com/ohbm/hackathon2020/blob/master/.github/ISSUE_TEMPLATE/handbooks/projects.md#chat).
+-   [ ] Video channel: Please create a video channel that will be used during the OHBM Brainhack and share it in your chat channel above. This can be an existing channel or a new one. For instance a [jitsi meet](https://meet.jit.si/) room, cf. [here](https://github.com/ohbm/hackathon2020/blob/master/.github/ISSUE_TEMPLATE/handbooks/projects.md#video-calls).
+-   [ ] Provide an image of your project for the OHBM brainhack website
+
```

Comment:
The other main two things missing at the moment are:

1. asking if the project would benefit from QMENTA support (and if it's the case invite them to get in touch with them through their slack);
2. asking if they have any suggestion for anyone co-leading the project in the "other" timeslot

Reply:
Can we elaborate on what support QMENTA could provide? Is there a link, perhaps?

Reply:
We can, of course, simply add the blurb from the save-the-date message.

Reply:
Let's do that, as the support is kind of open-ended, and the easiest choice is go and talk with them.